pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>decentralized-identity</groupId>
         <artifactId>uni-resolver</artifactId>
-        <version>0.1.0</version>
+        <version>0.3-SNAPSHOT</version>
```

Comment:
0.3.0-SNAPSHOT following SEMVER and Maven versioning

Reply:
Yes, however this is not my naming. This is the parent which has a version set by DIF (DanubeTech in this case). https://github.com/decentralized-identity/universal-resolver/blob/main/resolver/java/pom.xml

Reply:
Ah LOL. Yupz didn't notice that

Reply:
But then the 0.1-SNAPSHOT version is ours and has the same problem ;)

Reply:
Oh right, yes I can correct that one.

Reply:
Updated.